packages/p2p/src/components/payment-method-card/payment-method-card.tsx

```diff
@@ -16,34 +32,37 @@ const PaymentMethodCard = ({
     label = undefined,
     large,
     medium,
-    onClick = () => {},
-    onClickAdd = () => {},
+    onClick,
+    onClickAdd,
     payment_method,
     show_payment_method_name = true,
     small,
     style,
-}) => {
+}: TPaymentMethodCardProps) => {
     const { general_store, my_ads_store, my_profile_store } = useStores();
+    const { active_index } = general_store;
+    const { payment_method_ids } = my_ads_store;
+
     const { showModal } = useModalManagerContext();
     const { display_name, fields, icon, id } = payment_method || {};
-    const method = !is_add && display_name.replace(/\s|-/gm, '');
+    const method = !is_add && display_name?.replace(/\s|-/gm, '');
     const payment_account = fields?.account?.value;
     const payment_account_name = display_name;
     const payment_bank_name = fields?.bank_name?.value;
     const payment_name = fields?.name?.value;
 
-    const handleEditDeletPaymentMethod = e => {
-        if (e.target.value === 'delete') {
+    const handleEditDeletePaymentMethod = (event: { target: { value: string } }) => {
```

Comment:
```suggestion
    const handleEditDeletePaymentMethod = (event: Event) => {
```

🤔

Reply:
in the Dropdown component, for onChange prop, it was complaining that the typing should be `(e: { target: { name: string; value: string; }; }` hence why i kept it like that, if I change it to `Event` then itll complain :(